README.md

```diff
@@ -55,6 +65,23 @@ cd build
 ninja
 sudo ninja install
 ```
+# :computer: Contributing workflow
+Here’s how we suggest you go about proposing a change to this project:
+
+1. [Fork this project][fork] to your account.
+2. [Create a branch][branch] for the change you intend to make.
+3. Make your changes to your fork.
+4. [Send a pull request][pr] from your fork’s branch to our `master` branch
+
+
+[fork]: https://help.github.com/articles/fork-a-repo/
+[branch]: https://help.github.com/articles/creating-and-deleting-branches-within-your-repository
+[pr]: https://help.github.com/articles/using-pull-requests/
+
+# :tada: Special Thanks 
```

Comment:
Remove this from here. Don't want to maintain Authors.md and this simultaneously. Or just link the file here